coriolis/providers/replicator.py

```diff
@@ -795,27 +795,27 @@ def replicate_disks(self, source_volumes_info, backup_writer):
                 self._event_manager.progress_update(
                     "No new chunks to replicate for disk \"%s\" (%s)" % (
                         volume['disk_id'], devName))
```

Comment:
Why not just add the following two lines here?
```python
self._repl_state = curr_state
return self._repl_state
```

Reply:
Logically the same, but can do that too.